Rotation crash on „Video not available“ page

<!--
Oh no, a bug! It happens. Thanks for reporting an issue with NewPipe. To make it easier for us to help you please enter detailed information in the template we have provided below. If a section isn't relevant, just delete it, though it would be helpful to still provide as much detail as possible.
-->

<!-- IF YOU DON'T FILL IN THE TEMPLATE PROPERLY, YOUR ISSUE IS LIABLE TO BE CLOSED. If you feel tired/lazy right now, open your issue some other time. We'll wait. -->

<!-- The comments between these brackets won't show up in the submitted issue (as you can see in the Preview). -->

### Checklist
<!-- This checklist is COMPULSORY. The first box has been checked for you to show you how it is done. -->

- [x] I am using the latest version - 0.21.0 <!-- Check https://github.com/TeamNewPipe/NewPipe/releases -->
- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. <!-- Seriously, check. O_O -->
- [x] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [x] This issue contains only one bug. I will open one issue for every bug report I want to file.

### Steps to reproduce the bug
1. Open a video that's not available (invalid URL/private video/whatever)
2. Rotate the screen by 90°


<!-- If you can't cause the bug to show up again reliably (and hence don't have a proper set of steps to give us), please still try to give as many details as possible on how you think you encountered the bug. -->



### Actual behaviour
<!-- Tell us what happens with the steps given above. -->
NewPipe throws an UI error



### Expected behavior
<!-- Tell us what you expect to happen. -->
Just rotate the page by 90°



### Screenshots/Screen recordings
<!-- If applicable, add screenshots or a screen recording to help explain your problem. GitHub supports uploading them directly in the issue text box. If your file is too big for Github to accept, feel free to paste a link from an image/video hoster here instead. -->

<!-- DON'T POST SCREENSHOTS OF THE ERROR PAGE. Use the buttons given on the error page to paste the error as text in the Logs section below. -->

https://user-images.githubusercontent.com/62775760/112752430-3c736900-8fd3-11eb-9de1-59a109b83e4e.mp4




### Logs
<!-- If your bug includes a crash (where you're shown the Error Report page with a bunch of info), tap on "Copy formatted report" at the bottom and paste it here: -->

<!-- That's right, here! -->
## Exception
* __User Action:__ ui error
* __Request:__ ACRA report
* __Content Country:__ US
* __Content Language:__ en-US
* __App Language:__ en_US
* __Service:__ none
* __Version:__ 0.21.0
* __OS:__ Linux Android 11 - 30
<details><summary><b>Crash log </b></summary><p>

```
java.lang.RuntimeException: Unable to start activity ComponentInfo{org.schabi.newpipe/org.schabi.newpipe.MainActivity}: androidx.fragment.app.Fragment$InstantiationException: Unable to instantiate fragment org.schabi.newpipe.fragments.EmptyFragment: could not find Fragment constructor
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3792)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3968)
	at android.app.ActivityThread.handleRelaunchActivityInner(ActivityThread.java:5957)
	at android.app.ActivityThread.handleRelaunchActivity(ActivityThread.java:5863)
	at android.app.servertransaction.ActivityRelaunchItem.execute(ActivityRelaunchItem.java:69)
	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2307)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loop(Looper.java:246)
	at android.app.ActivityThread.main(ActivityThread.java:8506)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:602)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1130)
Caused by: androidx.fragment.app.Fragment$InstantiationException: Unable to instantiate fragment org.schabi.newpipe.fragments.EmptyFragment: could not find Fragment constructor
	at androidx.fragment.app.Fragment.instantiate(Fragment.java:563)
	at androidx.fragment.app.FragmentContainer.instantiate(FragmentContainer.java:57)
	at androidx.fragment.app.FragmentManager$3.instantiate(FragmentManager.java:390)
	at androidx.fragment.app.FragmentStateManager.<init>(FragmentStateManager.java:74)
	at androidx.fragment.app.FragmentManager.restoreSaveState(FragmentManager.java:2454)
	at androidx.fragment.app.Fragment.restoreChildFragmentState(Fragment.java:1706)
	at androidx.fragment.app.Fragment.onCreate(Fragment.java:1683)
	at org.schabi.newpipe.BaseFragment.onCreate(BaseFragment.java:55)
	at org.schabi.newpipe.fragments.detail.VideoDetailFragment.onCreate(VideoDetailFragment.java:279)
	at androidx.fragment.app.Fragment.performCreate(Fragment.java:2684)
	at androidx.fragment.app.FragmentStateManager.create(FragmentStateManager.java:280)
	at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1175)
	at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1356)
	at androidx.fragment.app.FragmentManager.moveFragmentToExpectedState(FragmentManager.java:1434)
	at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1497)
	at androidx.fragment.app.FragmentManager.dispatchStateChange(FragmentManager.java:2625)
	at androidx.fragment.app.FragmentManager.dispatchCreate(FragmentManager.java:2571)
	at androidx.fragment.app.FragmentController.dispatchCreate(FragmentController.java:236)
	at androidx.fragment.app.FragmentActivity.onCreate(FragmentActivity.java:315)
	at androidx.appcompat.app.AppCompatActivity.onCreate(AppCompatActivity.java:115)
	at org.schabi.newpipe.MainActivity.onCreate(MainActivity.java:139)
	at android.app.Activity.performCreate(Activity.java:8198)
	at android.app.Activity.performCreate(Activity.java:8182)
	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1309)
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3765)
	... 13 more
Caused by: java.lang.NoSuchMethodException: org.schabi.newpipe.fragments.EmptyFragment.<init> []
	at java.lang.Class.getConstructor0(Class.java:2332)
	at java.lang.Class.getConstructor(Class.java:1728)
	at androidx.fragment.app.Fragment.instantiate(Fragment.java:548)
	... 37 more

```
</details>
<hr>




<!-- Please fill this out when you do not provide a log generate by NewPipe -->

### Device info

 - Android version/Custom ROM version: Android 11 R, Stock ROM
 - Device model: Samsung Galaxy Note 10+

I was seeing the same on 0.21.1 so I went to try out v0.21.2_RC2 from #6052 to see if this was resolved already but it looks like it got "worse", as it doesn't even open the builtin crash reporter for me, just closes the app.
Seeing this in the logs I'm wondering though if this may be intended behavior and the crash log is automatically submitted?
```
04-21 23:53:24.822 27150 29836 I ACRA    : Sending report /data/user/0/org.schabi.newpipe.debug.release0212/app_ACRA-approved/2021-04-21T23:53:24.736+02:00.stacktrace
```

I can reproduce it with multiple available videos, including https://youtu.be/dQw4w9WgXcQ, just by rotating before the video itself is loaded.

demo:

https://user-images.githubusercontent.com/3694534/115626661-1bfeac00-a2fe-11eb-9a4e-38bb657b4c8c.mp4

<details><summary><b>Crash log (from logcat)</b></summary><p>

```
04-21 23:53:24.691 29734 29734 D AndroidRuntime: Shutting down VM
04-21 23:53:24.693 29734 29734 E AndroidRuntime: FATAL EXCEPTION: main
04-21 23:53:24.693 29734 29734 E AndroidRuntime: Process: org.schabi.newpipe.debug.release0212, PID: 29734
04-21 23:53:24.693 29734 29734 E AndroidRuntime: java.lang.RuntimeException: Unable to start activity ComponentInfo{org.schabi.newpipe.debug.release0212/org.schabi.newpipe.MainActivity}: androidx.fragment.app.Fragment$InstantiationException: Unable to instantiate fragment org.schabi.newpipe.fragments.EmptyFragment: could not find Fragment constructor
04-21 23:53:24.693 29734 29734 E AndroidRuntime: 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3431)
04-21 23:53:24.693 29734 29734 E AndroidRuntime: 	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3595)
04-21 23:53:24.693 29734 29734 E AndroidRuntime: 	at android.app.ActivityThread.handleRelaunchActivityInner(ActivityThread.java:5451)
04-21 23:53:24.693 29734 29734 E AndroidRuntime: 	at android.app.ActivityThread.handleRelaunchActivity(ActivityThread.java:5357)
04-21 23:53:24.693 29734 29734 E AndroidRuntime: 	at android.app.servertransaction.ActivityRelaunchItem.execute(ActivityRelaunchItem.java:69)
04-21 23:53:24.693 29734 29734 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
04-21 23:53:24.693 29734 29734 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
04-21 23:53:24.693 29734 29734 E AndroidRuntime: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2066)
04-21 23:53:24.693 29734 29734 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:106)
04-21 23:53:24.693 29734 29734 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:223)
04-21 23:53:24.693 29734 29734 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:7660)
04-21 23:53:24.693 29734 29734 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
04-21 23:53:24.693 29734 29734 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
04-21 23:53:24.693 29734 29734 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)
04-21 23:53:24.693 29734 29734 E AndroidRuntime: Caused by: androidx.fragment.app.Fragment$InstantiationException: Unable to instantiate fragment org.schabi.newpipe.fragments.EmptyFragment: could not find Fragment constructor
04-21 23:53:24.693 29734 29734 E AndroidRuntime: 	at androidx.fragment.app.Fragment.instantiate(Fragment.java:563)
04-21 23:53:24.693 29734 29734 E AndroidRuntime: 	at androidx.fragment.app.FragmentContainer.instantiate(FragmentContainer.java:57)
04-21 23:53:24.693 29734 29734 E AndroidRuntime: 	at androidx.fragment.app.FragmentManager$3.instantiate(FragmentManager.java:390)
04-21 23:53:24.693 29734 29734 E AndroidRuntime: 	at androidx.fragment.app.FragmentStateManager.<init>(FragmentStateManager.java:74)
04-21 23:53:24.693 29734 29734 E AndroidRuntime: 	at androidx.fragment.app.FragmentManager.restoreSaveState(FragmentManager.java:2454)
04-21 23:53:24.693 29734 29734 E AndroidRuntime: 	at androidx.fragment.app.Fragment.restoreChildFragmentState(Fragment.java:1706)
04-21 23:53:24.693 29734 29734 E AndroidRuntime: 	at androidx.fragment.app.Fragment.onCreate(Fragment.java:1683)
04-21 23:53:24.693 29734 29734 E AndroidRuntime: 	at org.schabi.newpipe.BaseFragment.onCreate(BaseFragment.java:55)
04-21 23:53:24.693 29734 29734 E AndroidRuntime: 	at org.schabi.newpipe.fragments.detail.VideoDetailFragment.onCreate(VideoDetailFragment.java:279)
04-21 23:53:24.693 29734 29734 E AndroidRuntime: 	at androidx.fragment.app.Fragment.performCreate(Fragment.java:2684)
04-21 23:53:24.693 29734 29734 E AndroidRuntime: 	at androidx.fragment.app.FragmentStateManager.create(FragmentStateManager.java:280)
04-21 23:53:24.693 29734 29734 E AndroidRuntime: 	at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1175)
04-21 23:53:24.693 29734 29734 E AndroidRuntime: 	at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1356)
04-21 23:53:24.693 29734 29734 E AndroidRuntime: 	at androidx.fragment.app.FragmentManager.moveFragmentToExpectedState(FragmentManager.java:1434)
04-21 23:53:24.693 29734 29734 E AndroidRuntime: 	at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1497)
04-21 23:53:24.693 29734 29734 E AndroidRuntime: 	at androidx.fragment.app.FragmentManager.dispatchStateChange(FragmentManager.java:2625)
04-21 23:53:24.693 29734 29734 E AndroidRuntime: 	at androidx.fragment.app.FragmentManager.dispatchCreate(FragmentManager.java:2571)
04-21 23:53:24.693 29734 29734 E AndroidRuntime: 	at androidx.fragment.app.FragmentController.dispatchCreate(FragmentController.java:236)
04-21 23:53:24.693 29734 29734 E AndroidRuntime: 	at androidx.fragment.app.FragmentActivity.onCreate(FragmentActivity.java:315)
04-21 23:53:24.693 29734 29734 E AndroidRuntime: 	at androidx.appcompat.app.AppCompatActivity.onCreate(AppCompatActivity.java:115)
04-21 23:53:24.693 29734 29734 E AndroidRuntime: 	at org.schabi.newpipe.MainActivity.onCreate(MainActivity.java:141)
04-21 23:53:24.693 29734 29734 E AndroidRuntime: 	at android.app.Activity.performCreate(Activity.java:8000)
04-21 23:53:24.693 29734 29734 E AndroidRuntime: 	at android.app.Activity.performCreate(Activity.java:7984)
04-21 23:53:24.693 29734 29734 E AndroidRuntime: 	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1309)
04-21 23:53:24.693 29734 29734 E AndroidRuntime: 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3404)
04-21 23:53:24.693 29734 29734 E AndroidRuntime: 	... 13 more
04-21 23:53:24.693 29734 29734 E AndroidRuntime: Caused by: java.lang.NoSuchMethodException: org.schabi.newpipe.fragments.EmptyFragment.<init> []
04-21 23:53:24.693 29734 29734 E AndroidRuntime: 	at java.lang.Class.getConstructor0(Class.java:2332)
04-21 23:53:24.693 29734 29734 E AndroidRuntime: 	at java.lang.Class.getConstructor(Class.java:1728)
04-21 23:53:24.693 29734 29734 E AndroidRuntime: 	at androidx.fragment.app.Fragment.instantiate(Fragment.java:548)
04-21 23:53:24.693 29734 29734 E AndroidRuntime: 	... 37 more
04-21 23:53:24.694 29734 29734 E ACRA    : ACRA caught a RuntimeException for org.schabi.newpipe.debug.release0212
04-21 23:53:24.694 29734 29734 E ACRA    : java.lang.RuntimeException: Unable to start activity ComponentInfo{org.schabi.newpipe.debug.release0212/org.schabi.newpipe.MainActivity}: androidx.fragment.app.Fragment$InstantiationException: Unable to instantiate fragment org.schabi.newpipe.fragments.EmptyFragment: could not find Fragment constructor
04-21 23:53:24.694 29734 29734 E ACRA    : 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3431)
04-21 23:53:24.694 29734 29734 E ACRA    : 	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3595)
04-21 23:53:24.694 29734 29734 E ACRA    : 	at android.app.ActivityThread.handleRelaunchActivityInner(ActivityThread.java:5451)
04-21 23:53:24.694 29734 29734 E ACRA    : 	at android.app.ActivityThread.handleRelaunchActivity(ActivityThread.java:5357)
04-21 23:53:24.694 29734 29734 E ACRA    : 	at android.app.servertransaction.ActivityRelaunchItem.execute(ActivityRelaunchItem.java:69)
04-21 23:53:24.694 29734 29734 E ACRA    : 	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
04-21 23:53:24.694 29734 29734 E ACRA    : 	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
04-21 23:53:24.694 29734 29734 E ACRA    : 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2066)
04-21 23:53:24.694 29734 29734 E ACRA    : 	at android.os.Handler.dispatchMessage(Handler.java:106)
04-21 23:53:24.694 29734 29734 E ACRA    : 	at android.os.Looper.loop(Looper.java:223)
04-21 23:53:24.694 29734 29734 E ACRA    : 	at android.app.ActivityThread.main(ActivityThread.java:7660)
04-21 23:53:24.694 29734 29734 E ACRA    : 	at java.lang.reflect.Method.invoke(Native Method)
04-21 23:53:24.694 29734 29734 E ACRA    : 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
04-21 23:53:24.694 29734 29734 E ACRA    : 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)
04-21 23:53:24.694 29734 29734 E ACRA    : Caused by: androidx.fragment.app.Fragment$InstantiationException: Unable to instantiate fragment org.schabi.newpipe.fragments.EmptyFragment: could not find Fragment constructor
04-21 23:53:24.694 29734 29734 E ACRA    : 	at androidx.fragment.app.Fragment.instantiate(Fragment.java:563)
04-21 23:53:24.694 29734 29734 E ACRA    : 	at androidx.fragment.app.FragmentContainer.instantiate(FragmentContainer.java:57)
04-21 23:53:24.694 29734 29734 E ACRA    : 	at androidx.fragment.app.FragmentManager$3.instantiate(FragmentManager.java:390)
04-21 23:53:24.694 29734 29734 E ACRA    : 	at androidx.fragment.app.FragmentStateManager.<init>(FragmentStateManager.java:74)
04-21 23:53:24.694 29734 29734 E ACRA    : 	at androidx.fragment.app.FragmentManager.restoreSaveState(FragmentManager.java:2454)
04-21 23:53:24.694 29734 29734 E ACRA    : 	at androidx.fragment.app.Fragment.restoreChildFragmentState(Fragment.java:1706)
04-21 23:53:24.694 29734 29734 E ACRA    : 	at androidx.fragment.app.Fragment.onCreate(Fragment.java:1683)
04-21 23:53:24.694 29734 29734 E ACRA    : 	at org.schabi.newpipe.BaseFragment.onCreate(BaseFragment.java:55)
04-21 23:53:24.694 29734 29734 E ACRA    : 	at org.schabi.newpipe.fragments.detail.VideoDetailFragment.onCreate(VideoDetailFragment.java:279)
04-21 23:53:24.694 29734 29734 E ACRA    : 	at androidx.fragment.app.Fragment.performCreate(Fragment.java:2684)
04-21 23:53:24.694 29734 29734 E ACRA    : 	at androidx.fragment.app.FragmentStateManager.create(FragmentStateManager.java:280)
04-21 23:53:24.694 29734 29734 E ACRA    : 	at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1175)
04-21 23:53:24.694 29734 29734 E ACRA    : 	at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1356)
04-21 23:53:24.694 29734 29734 E ACRA    : 	at androidx.fragment.app.FragmentManager.moveFragmentToExpectedState(FragmentManager.java:1434)
04-21 23:53:24.694 29734 29734 E ACRA    : 	at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1497)
04-21 23:53:24.694 29734 29734 E ACRA    : 	at androidx.fragment.app.FragmentManager.dispatchStateChange(FragmentManager.java:2625)
04-21 23:53:24.694 29734 29734 E ACRA    : 	at androidx.fragment.app.FragmentManager.dispatchCreate(FragmentManager.java:2571)
04-21 23:53:24.694 29734 29734 E ACRA    : 	at androidx.fragment.app.FragmentController.dispatchCreate(FragmentController.java:236)
04-21 23:53:24.694 29734 29734 E ACRA    : 	at androidx.fragment.app.FragmentActivity.onCreate(FragmentActivity.java:315)
04-21 23:53:24.694 29734 29734 E ACRA    : 	at androidx.appcompat.app.AppCompatActivity.onCreate(AppCompatActivity.java:115)
04-21 23:53:24.694 29734 29734 E ACRA    : 	at org.schabi.newpipe.MainActivity.onCreate(MainActivity.java:141)
04-21 23:53:24.694 29734 29734 E ACRA    : 	at android.app.Activity.performCreate(Activity.java:8000)
04-21 23:53:24.694 29734 29734 E ACRA    : 	at android.app.Activity.performCreate(Activity.java:7984)
04-21 23:53:24.694 29734 29734 E ACRA    : 	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1309)
04-21 23:53:24.694 29734 29734 E ACRA    : 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3404)
04-21 23:53:24.694 29734 29734 E ACRA    : 	... 13 more
04-21 23:53:24.694 29734 29734 E ACRA    : Caused by: java.lang.NoSuchMethodException: org.schabi.newpipe.fragments.EmptyFragment.<init> []
04-21 23:53:24.694 29734 29734 E ACRA    : 	at java.lang.Class.getConstructor0(Class.java:2332)
04-21 23:53:24.694 29734 29734 E ACRA    : 	at java.lang.Class.getConstructor(Class.java:1728)
04-21 23:53:24.694 29734 29734 E ACRA    : 	at androidx.fragment.app.Fragment.instantiate(Fragment.java:548)
04-21 23:53:24.694 29734 29734 E ACRA    : 	... 37 more
```
</details>
<hr>

This crash apparently also happens before a video loads.
1. Share video to newpipe video mode
2. Rotate device before video url is parsed completely and loaded
3. App crashes: 

________

Crash when rotate device before video loads when sharing
### Steps to reproduce the bug
<!--
1. Go to '...'
2. Press on '....'
3. Swipe down to '....'
-->
1. Open youtube video in browser
2. Share video to newpipe video mode
3. Rotate device before video loads (Bad connection makes it worse as it takes longer to load video giving it more time to crash, as if the video loads before rotating app does not crash)
4. App crashes when you rotate device before newpipe   parses and loads youtube video
<!-- If you can't cause the bug to show up again reliably (and hence don't have a proper set of steps to give us), please still try to give as many details as possible on how you think you encountered the bug. -->



### Actual behaviour
<!-- Tell us what happens with the steps given above. -->
App crashes


### Expected behavior
<!-- Tell us what you expect to happen. -->
Switch to landscape mode without crashing


### Screenshots/Screen recordings
<!-- If applicable, add screenshots or a screen recording to help explain your problem. GitHub supports uploading them directly in the issue text box. If your file is too big for Github to accept, feel free to paste a link from an image/video hoster here instead. -->

<!-- DON'T POST SCREENSHOTS OF THE ERROR PAGE. Use the buttons given on the error page to paste the error as text in the Logs section below. -->



### Logs
<!-- If your bug includes a crash (where you're shown the Error Report page with a bunch of info), tap on "Copy formatted report" at the bottom and paste it here: -->

<!-- That's right, here! -->

## Exception
* __User Action:__ ui error
* __Request:__ ACRA report
* __Content Country:__ UM
* __Content Language:__ en-GB
* __App Language:__ en_GB
* __Service:__ none
* __Version:__ 0.21.1
* __OS:__ Linux Android 10 - 29
<details><summary><b>Crash log </b></summary><p>

```
java.lang.RuntimeException: Unable to start activity ComponentInfo{org.schabi.newpipe/org.schabi.newpipe.MainActivity}: androidx.fragment.app.Fragment$InstantiationException: Unable to instantiate fragment org.schabi.newpipe.fragments.EmptyFragment: could not find Fragment constructor
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3270)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3409)
	at android.app.ActivityThread.handleRelaunchActivityInner(ActivityThread.java:5279)
	at android.app.ActivityThread.handleRelaunchActivity(ActivityThread.java:5187)
	at android.app.servertransaction.ActivityRelaunchItem.execute(ActivityRelaunchItem.java:69)
	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2016)
	at android.os.Handler.dispatchMessage(Handler.java:107)
	at android.os.Looper.loop(Looper.java:214)
	at android.app.ActivityThread.main(ActivityThread.java:7356)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:491)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:940)
Caused by: androidx.fragment.app.Fragment$InstantiationException: Unable to instantiate fragment org.schabi.newpipe.fragments.EmptyFragment: could not find Fragment constructor
	at androidx.fragment.app.Fragment.instantiate(Fragment.java:563)
	at androidx.fragment.app.FragmentContainer.instantiate(FragmentContainer.java:57)
	at androidx.fragment.app.FragmentManager$3.instantiate(FragmentManager.java:390)
	at androidx.fragment.app.FragmentStateManager.<init>(FragmentStateManager.java:74)
	at androidx.fragment.app.FragmentManager.restoreSaveState(FragmentManager.java:2454)
	at androidx.fragment.app.Fragment.restoreChildFragmentState(Fragment.java:1706)
	at androidx.fragment.app.Fragment.onCreate(Fragment.java:1683)
	at org.schabi.newpipe.BaseFragment.onCreate(BaseFragment.java:55)
	at org.schabi.newpipe.fragments.detail.VideoDetailFragment.onCreate(VideoDetailFragment.java:279)
	at androidx.fragment.app.Fragment.performCreate(Fragment.java:2684)
	at androidx.fragment.app.FragmentStateManager.create(FragmentStateManager.java:280)
	at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1175)
	at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1356)
	at androidx.fragment.app.FragmentManager.moveFragmentToExpectedState(FragmentManager.java:1434)
	at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1497)
	at androidx.fragment.app.FragmentManager.dispatchStateChange(FragmentManager.java:2625)
	at androidx.fragment.app.FragmentManager.dispatchCreate(FragmentManager.java:2571)
	at androidx.fragment.app.FragmentController.dispatchCreate(FragmentController.java:236)
	at androidx.fragment.app.FragmentActivity.onCreate(FragmentActivity.java:315)
	at androidx.appcompat.app.AppCompatActivity.onCreate(AppCompatActivity.java:115)
	at org.schabi.newpipe.MainActivity.onCreate(MainActivity.java:139)
	at android.app.Activity.performCreate(Activity.java:7824)
	at android.app.Activity.performCreate(Activity.java:7813)
	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1306)
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3245)
	... 13 more
Caused by: java.lang.NoSuchMethodException: org.schabi.newpipe.fragments.EmptyFragment.<init> []
	at java.lang.Class.getConstructor0(Class.java:2332)
	at java.lang.Class.getConstructor(Class.java:1728)
	at androidx.fragment.app.Fragment.instantiate(Fragment.java:548)
	... 37 more

```
</details>
<hr>


<!-- Please fill this out when you do not provide a log generate by NewPipe -->

### Device info

 - Android version/Custom ROM version: Android 10 | LineageOS

--- a/app/src/main/java/org/schabi/newpipe/fragments/EmptyFragment.java
+++ b/app/src/main/java/org/schabi/newpipe/fragments/EmptyFragment.java
@@ -11,15 +11,20 @@ import org.schabi.newpipe.BaseFragment;
 import org.schabi.newpipe.R;
 
 public class EmptyFragment extends BaseFragment {
-    final boolean showMessage;
+    private static final String SHOW_MESSAGE = "SHOW_MESSAGE";
 
-    public EmptyFragment(final boolean showMessage) {
-        this.showMessage = showMessage;
+    public static final EmptyFragment newInstance(final boolean showMessage) {
+        final EmptyFragment emptyFragment = new EmptyFragment();
+        final Bundle bundle = new Bundle(1);
+        bundle.putBoolean(SHOW_MESSAGE, showMessage);
+        emptyFragment.setArguments(bundle);
+        return emptyFragment;
     }
 
     @Override
     public View onCreateView(final LayoutInflater inflater, @Nullable final ViewGroup container,
                              final Bundle savedInstanceState) {
+        final boolean showMessage = getArguments().getBoolean(SHOW_MESSAGE);
         final View view = inflater.inflate(R.layout.fragment_empty, container, false);
         view.findViewById(R.id.empty_state_view).setVisibility(
                 showMessage ? View.VISIBLE : View.GONE);

--- a/app/src/main/java/org/schabi/newpipe/fragments/detail/VideoDetailFragment.java
+++ b/app/src/main/java/org/schabi/newpipe/fragments/detail/VideoDetailFragment.java
@@ -929,20 +929,20 @@ public final class VideoDetailFragment
 
         if (showRelatedItems && binding.relatedItemsLayout == null) {
             // temp empty fragment. will be updated in handleResult
-            pageAdapter.addFragment(new EmptyFragment(false), RELATED_TAB_TAG);
+            pageAdapter.addFragment(EmptyFragment.newInstance(false), RELATED_TAB_TAG);
             tabIcons.add(R.drawable.ic_art_track);
             tabContentDescriptions.add(R.string.related_items_tab_description);
         }
 
         if (showDescription) {
             // temp empty fragment. will be updated in handleResult
-            pageAdapter.addFragment(new EmptyFragment(false), DESCRIPTION_TAB_TAG);
+            pageAdapter.addFragment(EmptyFragment.newInstance(false), DESCRIPTION_TAB_TAG);
             tabIcons.add(R.drawable.ic_description);
             tabContentDescriptions.add(R.string.description_tab_description);
         }
 
         if (pageAdapter.getCount() == 0) {
-            pageAdapter.addFragment(new EmptyFragment(true), EMPTY_TAB_TAG);
+            pageAdapter.addFragment(EmptyFragment.newInstance(true), EMPTY_TAB_TAG);
         }
         pageAdapter.notifyDataSetUpdate();
 

